components/base/Input.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <input class="input" v-model="model" />
```

Comment:
Ah, I was half hoping no one would mention. I had issues doing that while also having `v-model` properly setup. You'd need a way to filter out the `value` prop from `$attrs` and `$listeners` otherwise the `v-model` returns an event object as apposed to the actual data. I may be going about it wrong however. I can look into getting that added.

Reply:
Opened ticket #32 for this